### PR TITLE
Plans: Rework feature comparison table

### PR DESCRIPTION
**Pull request by cwickert**
I have slightly improved the PAYG vs. BYOS feature comparison. Please check and if you have any ideas for more rows, please let me know.

feature-comparion-table

### CODE REVIEW

**Comment by kvanderveer** [comment]
How about heading = “customer relationship” and “with cloud provider” for payg and “with cloud provider and SUSE” for byos?

File: xml/cha_images.xml [hunk 1]
```diff
@@ -156,54 +163,87 @@
      <row>
       <entry>
        <para>
-        Instances are ready to use
+        Customer
```
Comment (kvanderveer) [inline xml/cha_images.xml]: How about heading = “customer relationship” and “with cloud provider” for payg and “with cloud provider and SUSE” for byos?

**Review by kvanderveer — CHANGES_REQUESTED** [review]

File: xml/cha_images.xml (hunk 1 above)
Comment (cwickert) [reply xml/cha_images.xml]: Thanks, applied in latest commit.

**Comment by cwickert** [comment]
Should we make LTSS a separate row to emphasize the difference? Or are there products where PAYG can get LTSS, e.g. SAP?

**Comment by rjschwei** [comment]
> Should we make LTSS a separate row to emphasize the difference? Or are there products where PAYG can get LTSS, e.g. SAP?

SLES For SAP has ESPOS,  while conceptually the same as LTSS, ESPOS is included in the product (SLES For SAP) while LTSS is an add on product. Also note that we will soon announce a feature where BYOS users can use the update infrastructure. Therefore the table content needs some modification.

**Comment by cwickert** [comment]
@rjschwei how about this?
Row title: "LTSS/ESPOS"
PAYG: "only if included in image product"
BYOS: "available"

We could then link to https://www.playmobil.de/hipster-mit-e-roller/70873.html for further information.

**Review by kvanderveer — APPROVED** [review]

**Comment by rjschwei** [comment]
> @rjschwei how about this? Row title: "LTSS/ESPOS" PAYG: "only if included in image product" BYOS: "available"
> 
> We could then link to https://www.playmobil.de/hipster-mit-e-roller/70873.html for further information.

Sounds reasonable to me, not sure where the playmobil link comes into play, but a link to the LTSS product page would be helpful.

Also note that we'll need some rework w.r.t. registration. The "BYOS to update infrastructure" setup will be announced this week. Meaning this could be merged and then a follow up PR could be created to make the adjustments fo rthe new feature.

**Comment by cwickert** [comment]
> not sure where the playmobil link comes into play, but a link to the LTSS product page would be helpful.

Sorry, copy & paste fail (curse of having two clipboards), I wanted to link https://www.suse.com/lifecycle instead. If this works, please approve this one so I can merge it before we do the next PR about the new update infra.

**Review by rjschwei — APPROVED** [review]

**Pull request merged**